specification/EnterpriseKnowledgeGraph/resource-manager/readme.md

```diff
@@ -22,8 +22,8 @@ Please also specify `--azure-libraries-for-java=<path to the root directory of y
 
 ``` yaml $(tag) == 'package-2018-12-03' && $(java) && $(multiapi)
 java:
-  namespace: com.microsoft.azure.management.EnterpriseKnowledgeGraphservice.v2018-12-03
```

Comment:
@jianghaolu could this be a breaking change?

Reply:
Nothing released. This is fine.